lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -312,6 +317,63 @@ def cmd_log(*args)
     end
   end
 
+  #
+  # Interact with framework's service manager
+  #
+  def cmd_servicemanager(*args)
```

Comment:
There was discussion over naming this to something like `moduleservices`. I can throw this behind a feature flag if we decide not to make it a 'public' api.